docs/README.adoc

```diff
@@ -446,8 +446,8 @@ substrate-trie
 sr-api, sr-io, sr-primitives, sr-sandbox, sr-std, sr-version
 * Substrate Runtime Module Library (SRML)
```

Comment:
This also needs renaming, I guess?

Reply:
There are actually a number of docs which still need to be fixed

Reply:
I patched this, but we should do a docs pass at another time (so we dont block polkadot release)